pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -139,6 +154,23 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 	return reconcile.Result{}, nil
 }
 
+// migrateLabels is a temporary function that will replace deprecated labels with non-deprecated labels
+// Deprecated: Remove this function once all deprecated labels have been migrated
+func (r *ReconcileMasterUserRecord) migrateLabels(mur *toolchainv1alpha1.MasterUserRecord) (bool, error) {
```

Comment:
Btw, there is already a function for migrating MURs: https://github.com/codeready-toolchain/host-operator/blob/master/pkg/controller/usersignup/masteruserrecord.go#L9
but it's fine keeping it as it is, since you are going to remove it anyway...